docs/examples/jobs/httpd.yaml

```diff
@@ -0,0 +1,8 @@
+name: httpd
```

Comment:
What is httpd.yaml file here?
Is there a relationship to job.yaml?

Reply:
removed